e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -21,22 +21,48 @@ describe('team management', () => {
   const describeIAMV2 = iamVersion.match(/v2/) ? describe : describe.skip;
   const describeProjectsEnabled = iamVersion === 'v2.1' ? describe : describe.skip;
 
+  before(() => {
+    cy.adminLogin('/settings/teams').then(() => {
+      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      adminToken = admin.id_token;
+      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+      cy.cleanupTeamsByDescriptionPrefix(adminToken, cypressPrefix);
+      cy.request({
+        auth: { bearer: adminToken },
+        method: 'POST',
+        url: '/apis/iam/v2beta/projects',
+        body: {
+          id: project1ID,
+          name: project1Name
+        }
+      });
+      cy.request({
+        auth: { bearer: adminToken },
+        method: 'POST',
+        url: '/apis/iam/v2beta/projects',
+        body: {
+          id: project2ID,
+          name: project2Name
+        }
+      });
+
+      // reload so we get projects in project filter
+      cy.reload(true);
+      cy.get('app-welcome-modal').invoke('hide');
+    });
+    cy.restoreStorage();
+  });
+
+  beforeEach(() => {
+    cy.restoreStorage();
+  });
+
   afterEach(() => {
     cy.saveStorage();
     cy.cleanupTeamsByDescriptionPrefix(adminToken, cypressPrefix);
   });
 
   context('no custom initial page state', () => {
```

Comment:
I've been meaning to ask, what is `context` and how is it being used in these tests? How is it different from `describe`?

Reply:
They are functionally exactly the same but they are used in different situations:

>>> The purpose of “describe” is to wrap a set of tests against one functionality while “context” is to wrap a set of tests against one functionality under the same state.

This article goes into good detail:

http://lmws.net/describe-vs-context-in-rspec

Reply:
If I have used it incorrectly somewhere feel free to call it out! I'm sure I have at some point.